src/imitation/policies/interactive.py

```diff
@@ -0,0 +1,148 @@
+"""Interactive policies that query the user for actions."""
+
+import abc
+import collections
+import typing
+
+import gym
+import matplotlib.pyplot as plt
+import numpy as np
+from stable_baselines3.common import vec_env
+
+import imitation.policies.base as base_policies
+from imitation.util import util
+
+
+class DiscreteInteractivePolicy(base_policies.NonTrainablePolicy, abc.ABC):
+    """Abstract class for interactive policies with discrete actions.
+
+    For each query, the observation is rendered and then the action is provided
+    as a keyboard input.
+    """
+
+    def __init__(
+        self,
+        observation_space: gym.Space,
+        action_space: gym.Space,
+        action_keys_names: collections.OrderedDict,
```

Comment:
Ideally, we'd have `OrderedDict[str, str]` as a type here. Note however that depending on the Python version, this would need to be imported either from `collections` or `typing`, so omitting it to avoid depending on a concrete version. Reference: https://stackoverflow.com/questions/41207128/how-do-i-specify-ordereddict-k-v-types-for-mypy-type-annotation#comment115837552_43583996

Reply:
We require Python >=3.8 for `imitation` -- does the problem arise in that (or later) version?

Reply:
Yes, the latest change was at 3.9 so 3.8 is not sufficient.